src/govuk/components/hide-this-page/_index.scss

```diff
@@ -0,0 +1,31 @@
+@import "../button/index";
+
+@include govuk-exports("govuk/component/hide-this-page") {
+  .govuk-hide-this-page {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100%;
+
+    @include govuk-media-query($from: tablet) {
+      display: inline-block;
+      position: sticky;
```

Comment:
Does this need to be progressively enhanced for browsers without support for `position: sticky`?

Reply:
It already is! (sort of). Currently on browser not supporting `sticky` it falls back to `fixed` found on line 5. There's a question about the tradeoff of having a button that follows the user but is outside the bounds of the width container and a button that doesn't follow the user but is easier to notice and access visually when it is on the screen. I'll revisit this with a designer once this has gone through some research.